docs/user/whats-new.asciidoc

```diff
@@ -14,20 +14,192 @@ Previous versions: {kibana-ref-all}/8.3/whats-new.html[8.3] | {kibana-ref-all}/8
 
 coming::[8.4.0]
 
+[discrete]
+[[highlights-8.4-release-docs]]
+=== Elastic release docs now in one place
+Want to learn about all the latest Elastic features and updates?
+We’ve made it easy for you to find everything you need to know with
+the new
+https://www.elastic.co/guide/en/welcome-to-elastic/current/new.html[Check out the latest from Elastic] page.
+Now you can access all release information you need in a single location.
+
+[role="screenshot"]
+image:images/highlights-release-docs.png[New release docs page on elastic.co, width=75%]
```

Comment:
This image looks like the new release docs page includes only Observability. 

This is a better image:
<img width="560" alt="Screen Shot 2022-08-15 at 11 20 49 AM" src="https://user-images.githubusercontent.com/38990715/185459152-a6c32569-3a85-406d-a36c-5e57d3a9d8d0.png">